.github/workflows/pytest.yml

```diff
@@ -0,0 +1,31 @@
+name: Reusable WF - Pytests
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        required: true
+        type: string
+
+jobs:
+
+  test:
+    name: Pytest (${{ inputs.python-version }})
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python-version }}
+
+      - name: Run unit tests
+        run: |
+          python --version
+          pip install -r requirements.in
+          pip install -r requirements-dev.in
+          loglevel=${1:-INFO}
+          HOWSO_CONFIG=${HOWSO_CONFIG:-./config/latest-st-howso.yml}
```

Comment:
Was ST used in the original testing as well? I'm wondering if it should updated to MT...

Reply:
Oops, this line is actually outdated and should not be used at all.. no config needed 🙂

Reply:
@calebwherry test